.github/workflows/lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Lint
+
+# NOTE(mhayden): Restricting branches prevents jobs from being doubled since
+# a push to a pull request triggers two events.
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - master
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Install golangci-lint
+        run: curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.23.7
```

Comment:
Is there a way to install this using a package manager?

Reply:
I think it isn't. However, is it a better option? `curl` into pipe is a bit ugly and can be dangerous but not on a CI environment which just gets deleted after finished. Additionally, versioning of the lint tool is much easier (you know the story about Fedora and keeping just two versions of packages in the repos). I think I don't mind the current installation method.

There's option to use a pre-built container from dockerhub. This might a bit better but I kinda disliked the idea of downloading a whole container to do linting.

Reply:
I think the most important thing is to pin the version, which this seems to do, so I'm convinced.